### ISSUE TITLE
Link back to Telegram API in documentation

### ISSUE THREAD

**Issue by j-maas** [Leonqn/rutebot#11]
I really liked that the [submodules would link to the corresponding Telegram API method](https://github.com/Leonqn/rutebot/blob/master/src/requests.rs#L12). Do you think it's a good idea to add such links for all types?

E. g., for the [update]() response, we could add the link to the end of the description:

> This object represents an incoming update. At most one of the optional parameters can be present in any given update
>
> See [Update](https://core.telegram.org/bots/api#update) in the Telegram API documentation for more details.

I'm only unsure about types like [`ChatId`](https://github.com/Leonqn/rutebot/blob/master/src/requests.rs#L241), that totally make sense, but only implicitely exist in the Telegram API. But even those could refer to the documentation.

What do you think?

**Comment by Leonqn** [comment]
Nice idea!
Also i think that we should add such links to request structs, because this PR removes public modules and docs to them https://github.com/Leonqn/rutebot/pull/10
And types like ChatId can simply stay without "more details".

**Comment by j-maas** [comment]
Yes, exactly. I meant for both requests and responses. However, is there a way to automate this? Or did you manually copy each documentation piece?

**Comment by Leonqn** [comment]
Yes i did it manually :)

**Comment by j-maas** [comment]
I think it's easiest to do it little by little then. Doing one big PR that does all at once requires some time, and it will be prone to merge conflicts, if we touch those structs. Therefore, doing it piece by piece seems a good option.
